### PR TITLE
Adding Labeler workflow

### LINKED ISSUE #155: [CI] Add automatic GitHub issue labeler configuration for Cloud and MeshMap paths
#### Current Behavior
GitHub PRs are not auto labeled based on the docs sections that they affect.

#### Implementation
e.g. `content/en/meshmap/**/*` should be automatically labeled with `area/cloud-docs`

#### Acceptance Tests
Both Cloud and MeshMap sections are covered.

---
#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by surajvast1**
**Notes for Reviewers**

This PR fixes #155

**[Signed commits](../blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: .github/labeler.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+
+component/cloud-docs:
+  - label: bug
+    paths:
+      - "content/en/meshmap/**/*"
```
Comment (hamza-mohd) [inline .github/labeler.yml:5]: Why are you labeling meshmap content with `component-cloud-docs`? Don't.

**Review by hamza-mohd — CHANGES_REQUESTED** [review]

**Comment by hamza-mohd** [comment]
Add `area/ci` for .github

**Review by iArchitSharma — CHANGES_REQUESTED** [review]

File: .github/labeler.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+
+component/cloud-docs:
```
[the remark this answers is not in the document]
Comment (surajvast1) [reply .github/labeler.yml:2]: @iArchitSharma Sure I assumed both to be under a single label . 
I'll make the changes

**Comment by saurabh100ni** [comment]
Hi @surajvast1  ,
Thanks for your PR! I encourage you to share this in the Website Meeting tomorrow at 6:30 PM IST. Share your screen, and walk us through it. It's cool to take and share feedback. Check the [Doc](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit) here to include this PR.

You're welcome at the meeting

Check to [community calendar](https://layer5.io/community/calendar) for more details of meetings at Layers.

**Comment by surajvast1** [comment]
@iArchitSharma its updated and good to go

**Comment by iArchitSharma** [comment]
@Yashsharma1911 can you please check if it works correctly ?

File: .github/labeler.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (leecalcote) [reply .github/labeler.yml:5]: @surajvast1 don't include meshery-docs here. That is a separate site. Use meshmap for meshmap.

One more chance to get this right. If you don't this time time, let's close this PR and allow another contributor a turn.

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: .github/labeler.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (surajvast1) [reply .github/labeler.yml:2]: @leecalcote I guess there is a lack of communication here . I implemented it correctly before but as @iArchitSharma ask me to add component/meshmap-docs i changed it . Its updated now

**Comment by surajvast1** [comment]
I have tested the labeler workflow at my end it works perfectly. 
Test1) Updating a random file inside a code not in the mentioned path so it doesnot adds labeler https://github.com/surajvast1/docs/pull/17/files

Test2) Updating a file in this path content/en/cloud so it again does not adds labeler 
https://github.com/surajvast1/docs/pull/18/files

Test3) Updating a file in the path content/en/meshmap/**/*  as mentioned in the error .So you can see labeler here 
https://github.com/surajvast1/docs/pull/19

Please consider it ,apologies for the miscommunication 😊.

**Comment by leecalcote** [comment]
good deal. @iArchitSharma are we good to go?

**Comment by iArchitSharma** [comment]
@surajvast1 the request was to add the meshmap label when the meshmap docs are edited and add the cloud label when cloud docs are edited. Your pull request only adds the 'cloud' labeler to both."

**Comment by iArchitSharma** [comment]
@surajvast1 https://github.com/layer5io/docs/pull/224

**Comment by surajvast1** [comment]
Now you have added meshery-docs. I hope it maches the requirements.Thanks I got to learn a lot working on this .

**Comment by iArchitSharma** [comment]
@surajvast1 It is meshmap docs label which is different from meshery

**Pull request closed without merging**